tests/test_headers.py

```diff
@@ -164,11 +164,12 @@ def test_raw_headers(app):
         },
     )
 
-    assert request.raw_headers == (
-        b"Host: example.com\r\nAccept: */*\r\nAccept-Encoding: gzip, "
-        b"deflate\r\nConnection: keep-alive\r\nUser-Agent: "
-        b"Sanic-Testing\r\nFOO: bar"
-    )
+    assert b"Host: example.com" in request.raw_headers
+    assert b"Accept: */*" in request.raw_headers
+    assert b"Accept-Encoding: gzip, deflate" in request.raw_headers
+    assert b"Connection: keep-alive" in request.raw_headers
+    assert b"User-Agent: Sanic-Testing" in request.raw_headers
+    assert b"FOO: bar" in request.raw_headers
```

Comment:
Just to further constraint the possible matches...

```suggestion
    assert b"Host: example.com\r\n" in request.raw_headers
    assert b"Accept: */*\r\n" in request.raw_headers
    assert b"Accept-Encoding: gzip, deflate\r\n" in request.raw_headers
    assert b"Connection: keep-alive\r\n" in request.raw_headers
    assert b"User-Agent: Sanic-Testing\r\n" in request.raw_headers
    assert b"FOO: bar\r\n" in request.raw_headers
```